app/controllers/users_controller.rb

```diff
@@ -0,0 +1,57 @@
+class UsersController < ApplicationController
```

Comment:
does this controller + route make sense anymore now that we have all the devise stuff?